environment.yml

```diff
@@ -1,38 +1,53 @@
+# 1. Install a conda distribution.
+#    https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html
+# 2. Run: `mamba env create -f environment.yml`
+# 3. Activate: `conda activate spyglass`
+#
+# (lines intentionally left blank)
+#
+#
 name: spyglass
 channels:
   - conda-forge
   - defaults
+  # - pytorch # dlc-only
   - franklab
   - edeno
 dependencies:
-  - python>=3.9,<3.10
+  - bottleneck
+  # - cudatoolkit=11.3 # dlc-only
+  # - cupy # dlc-only
+  # - dask-cuda # dlc-only
+  # - ffmpeg # dlc-only
+  - ipympl
   - jupyterlab>=3.*
-  - pydotplus
-  - dask
+  # - libgcc # dlc-only
+  - matplotlib
+  - non_local_detector
+  - numpy<1.24
   - pip
   - position_tools
-  - numpy<1.24
-  - track_linearization>=2.3
+  - pybind11  # req by mountainsort4 -> isosplit5
+  - pydotplus
+  - pyfftw<=0.12.0  # ghostipy req. install from conda-forge for Mac ARM
+  - python>=3.9,<3.10
+  - pytorch<1.12.0
+  - replay_trajectory_classification
   - ripple_detection
-  - non_local_detector
-  - matplotlib
   - seaborn
-  - bottleneck
-  - ipympl
-  - tqdm
-  - pyfftw<=0.12.0  # used by ghostipy. install from conda-forge so that it works on Mac ARM processors
+  # - torchaudio # dlc-only
+  # - torchvision # dlc-only
+  - track_linearization>=2.3
+  # - trajectory_analysis_tools # dlc-only
```

Comment:
We can remove `trajectory_analysis_tools` and `replay_trajectory_classification` because these are superseded by `non_local_detector`